integration-scripts/src/fixtures/agreement_template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "initiators": [{ "type": "execagreement" }],
+  "tasks": [
+    {
+      "type": "HttpGet",
+      "params": { "get": "https://bitstamp.net/api/ticker/" }
+    },
+    { "type": "JsonParse", "params": { "path": ["last"] } },
+    { "type": "EthBytes32" },
+    {
+      "type": "EthTx",
+      "params": {
+        "address": "0x356a04bce728ba4c62a30294a55e6a8600a320b3",
+        "functionSelector": "0x609ff1bd"
+      }
+    }
+  ],
+  "payment": "1000000000000000000",
+  "expiration": 300,
+  "oracles": ["0x9CA9d2D5E04012C9Ed24C0e513C9bfAa4A2dD77f"],
```

Comment:
Should we change this to make it more clear which fields are templatized?

Reply:
Yeah, I think it would be valid to name it something like "{ORACLE_ADDRESS}" instead of a valid address